splunk apm readme update (#14)
signalfx/lambda-layer-versions: splunk-apm/README.md

    If you want to use this exporter, set these environment variables in your Lambda function code:
    ```
    OTEL_EXPORTER_OTLP_ENDPOINT: <Endpoint URL>
    ```

>     Splunk provides also token-authenticated `jaeger-thrift-splunk` exporter for customers that need to use that specific protocol. In order to use it, please set:

Reviewer: ```suggestion
    Splunk provides also a token-authenticated `jaeger-thrift-splunk` exporter for customers that need to use that specific protocol. To use it, set the following environment variables:
```

5. Logging
    
    Logging is controller with `OTEL_LAMBDA_LOG_LEVEL` environment variable. Default set to `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
    

>     Please note that enabling `DEBUG` will generate additional logs which may in turn increase AWS CloudWatch costs. 

Reviewer: ```suggestion
    > Enabling `DEBUG` generates additional logs, which may in turn increase AWS CloudWatch costs. 
```
